pkg/metrics.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright DDEV Technologies LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkg
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// Prometheus metrics for bot command invocations
+var (
+	CreatePreviewSiteCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "bot_create_preview_site_total",
+		Help: "Total number of create preview site command invocations",
+	}, []string{"bot", "status"})
```

Comment:
`"bot"` means the type of the bot, we run two, one as part of gitlab-webhook-server and second under github-operator.
`"status"` is one of the five statuses described below, to give us a better idea of how frequently our users struggle with the bot.